File: Python/2 - Condicionais/5 - troco_verificado.py
preco = float(input('Preço unitário do produto: '))
quantidade = int(input('Quantidade comprada: '))
dinheiro = float(input('Dinheiro recebido: '))
troco = dinheiro - quantidade * preco
if troco < 0:
    print(f'DINHEIRO INSUFICIENTE. FALTAM {quantidade * preco - dinheiro} REAIS')
else:
    print(f'TROCO = {troco}')

File: C/2 - Condicionais/5 - troco_verificado.c
#include <stdio.h>

int main() {
    
    double preco, quant, dinheiro, troco;

    printf("Preco unitario do produto: ");
    scanf("%lf", &preco);
    printf("Quantidade comprada: ");
    scanf("%lf", &quant);
    printf("Dinheiro recebido: ");
    scanf("%lf", &dinheiro);

    troco = dinheiro - preco * quant;

    if (troco > 0) {
        printf("TROCO = %.2lf", troco);
    }
    else {
        printf("DINHEIRO INSUFICIENTE. FALTAM %.2lf REAIS", preco * quant - dinheiro);
    }

    return 0;
}